Classes/View/PdfView.php

```diff
@@ -127,7 +127,21 @@ private function replaceStrings($content) {
             }
         }
 
-        return $content;
+        return preg_replace_callback('/\<a (.*?)>/', function ($hit) {
+
+            $completeATag = $hit[0];
+            $returnTag = preg_replace_callback('/href="(.*)#(.*)"/', function ($href) {
+
+                if (!empty($href[2]) &&
+                        ($href[1] === '/' . str_replace('&', '&amp;', GeneralUtility::getIndpEnv(TYPO3_SITE_SCRIPT)))
+                ) {
+                    return 'href="#' . $href[2] . '"';
+                }
+            }, $completeATag);
+
+            return $returnTag;
+
+        }, $content);
```

Comment:
:+1: on the other changes, but I think I'd prefer a separate pull request for the link-handling stuff.